app/controllers/admin/editions_controller.rb

```diff
@@ -234,6 +234,7 @@ def permitted_edition_attributes
         lead_organisation_ids: [],
```

Comment:
Again, not complaining, but this implies that no editionable content types can have associated roles. Definitely another chunk of Whitehall weirdness

Reply:
I think this just implies that the `roles_ids` should be an array, rather than a string or anything else.

Reply:
i more mean this commit in general, like you've had to add the role tagging functionality from scratch to editions because it wasn't there already. Basically I'm just thinking about code that can be removed once (if ever) all types are editionable

Reply:
Ah, yes, I see.

We could associate with role appointments (e.g. on speeches, where we want to say the name of the actual person who gave a speech and what their role was at the time), but that wouldn't work in this case as we want the organisation associated with the role (otherwise it'll need to be updated manually each time someone new is appointed).

I'm hoping this code may be needed for other things we could make editionable (e.g. normal organisations).